### PR TITLE
Updating sidebar

### PR DESCRIPTION
**Pull request by kathleenAMA**
These are the styles for the sidebar. Also added in formatting for the rest of the sidebar.scss file.

@kaytcampbell

### CODE REVIEW

**Comment by kaytcampbell** [comment]
:art: :art: :art:
But will need a dev review too.

**Comment by kathleenAMA** [comment]
:wave: for the devs

File: app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+@charset "UTF-8";
```
Comment (jbabe) [inline app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss]: @kathleenAMA can you move this under `vendor/assets/stylesheets/.keep`:

`git mv app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss vendor/assets/stylesheets/`

Comment (jbabe) [inline app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss]: basically anything that we didn't write (e.g. a library from someone else) we try to keep under the vendor folder

**Comment by jbabe** [comment]
:hankey:

File: app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss (hunk 1 above)
Comment (jbabe) [inline app/assets/stylesheets/ama_layout/webfonts/ss-symbolicons-block.scss]: after looking at other gems it doesn't look like they do this - so my previous comment can be ignored.

**Comment by jbabe** [comment]
This isn't within the scope of this PR but how does the navigation get built out? 

Does a member have one nav, a non-member a different nav, and a logged out user a different one too?

**Comment by jbabe** [comment]
:elephant: :elephant: :elephant:

**Comment by kathleenAMA** [comment]
@jbabe That's basically how it works. Each app has a yml file that decides what links are pulled in and if the side bar needs to have different states. I think we should revisit this at some point though because most apps don't have more than one state.

**Pull request merged**